pkg/ccl/streamingccl/streamingest/stream_ingestion_job.go

```diff
@@ -480,9 +480,10 @@ func maybeRevertToCutoverTimestamp(
 	// existed in the record at the point of the update rather the
 	// value that may be in the job record before the update.
 	var (
-		shouldRevertToCutover bool
-		cutoverTimestamp      hlc.Timestamp
-		spansToRevert         roachpb.Spans
+		shouldRevertToCutover  bool
+		cutoverTimestamp       hlc.Timestamp
+		originalSpansToRevert  roachpb.Spans
```

Comment:
the original can be a single span, right? the tenant span, so it will never have multiple spans. probably better to keep as a single span so that the reader would know better what's going on?

Reply:
Done.